doc/api/process.md

```diff
@@ -1041,6 +1041,33 @@ a code.
 Specifying a code to [`process.exit(code)`][`process.exit()`] will override any
 previous setting of `process.exitCode`.
 
+## process.getActiveHandles()
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Array}
+
+Returns an array containing all handles that are preventing the Node.js process
+from exiting.
+
+```js
+console.log(`Active handles: ${util.inspect(process.getActiveHandles())}`);
+```
```

Comment:
More information about the things that are in the array would be helpful. What kind of objects are they? What can the user basically do with them, ec

Reply:
+1, `util.inspect`ing them and then `console.log`ing does not seem to be a good example since `console.log` will `util.inspect` them anyway..

Reply:
I would like to now active handles open for a particular process id. How can I get it